libs-scala/resources-akka/src/main/scala/com/digitalasset/resources/akka/AkkaResourceOwnerFactories.scala

```diff
@@ -28,8 +28,8 @@ trait AkkaResourceOwnerFactories[Context] {
   def forCancellable[C <: Cancellable](acquire: () => C): AbstractResourceOwner[Context, C] =
     new CancellableResourceOwner(acquire)
 
-  def forSourceQueue[T](
-      queueGraph: RunnableGraph[SourceQueueWithComplete[T]]
-  )(implicit materializer: Materializer): AbstractResourceOwner[Context, SourceQueue[T]] =
-    new SourceQueueResourceOwner[T, Context](queueGraph)
+  def forBoundedSourceQueue[T](
```

Comment:
Was the backpressure mechanism never used by any other clients?

Reply:
Only by tests which I moved to the `BoundedSourceQueue`.